benchmark/collective/bert/train.py

```diff
@@ -225,13 +225,8 @@ def train(args):
 
     role = role_maker.PaddleCloudRoleMaker(is_collective=True)
     fleet.init(role)
-    print("fleet.node_num:", fleet.node_num())
-
-    if fleet.node_num() > 1:
-        os.environ["FLAGS_sync_nccl_allreduce"] = "0"
-        dist_strategy.nccl_comm_num = 3
-        dist_strategy.fuse_memory_size = 64 #MB
-        dist_strategy.use_hierarchical_allreduce = True
+    dist_strategy.nccl_comm_num = 3
+    dist_strategy.use_hierarchical_allreduce = True
```

Comment:
missing fuse_memory_size?

Reply:
`fuse_memory_size`  is a bug. Delete it from fleetapi.